src/tests/node.test.ts

```diff
@@ -6,7 +6,7 @@ describe('Node JS environment', () => {
     let failedAsPlanned: boolean;
 
     beforeEach(async () => {
-        await new Promise((resolve) => setTimeout(resolve, 500));
+        await new Promise((resolve) => setTimeout(resolve, 1500));
```

Comment:
Why the need to increase timeouts?

Reply:
On second thought, we could possibly change the memo or quantity to avoid duplicate transaction more reliably and without additional time during tests.

Reply:
So with it at 500ms, you were getting duplicate transactions?

Reply:
The web tests were getting duplicates at 1000ms, I increased that one and brought the node tests to the same amount.  Can't recall if I also got duplicates in node with 500.